Sources/BraintreePayPal/BTPayPalVaultErrorHandlingEditRequest.swift

```diff
@@ -1,24 +1,24 @@
 import Foundation
 
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
 /// Options for the PayPal edit funding instrument flow with retry on failed attempts
 ///
 /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-public struct BTPayPalVaultErrorHandlingEditRequest {
+public class BTPayPalVaultErrorHandlingEditRequest: BTPayPalVaultEditRequest {
 
-    private let editPayPalVaultID: String
-    private let riskCorrelationID: String
-    private let merchantAccountID: String?
+    let riskCorrelationID: String
```

Comment:
qq: should this be private as well?

Reply:
Great question, this is used internally in `BTPayPalClient` on line 493: `dataCollector.clientMetadataID(editRequest.riskCorrelationID)` so needs to be internal since it wouldn't be accessible there if it was private.